cmd/minikube/cmd/start.go

```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```

Comment:
I guess you need the config, but you still don't have to put it into the function which gets retried.